elements/sdk-depends/eos-shard.bst

```diff
@@ -2,6 +2,7 @@ kind: autotools
 
 build-depends:
 - freedesktop-sdk.bst:public-stacks/buildsystem-autotools.bst
+- sdk-build-depends/jasmine-gjs.bst
```

Comment:
Couldn't you "List Append" this in `elements/tests/sdk/libdmodel.bst`?

Reply:
Oh yeah, that would be better :) There are a couple of modules (eos-sdk and eos-knowledge-content-renderer) which refuse to build if jasmine isn't available: https://github.com/endlessm/eos-knowledge-content-renderer/blob/master/tests/meson.build#L7. Probably fixable, but for now I'll just leave jasmine-gjs.bst as a build dependency for regular builds over there.